doc/interfaces/gs_usb.rst

```diff
@@ -0,0 +1,30 @@
+.. _gs_usb:
+
+CAN driver based on WCID for Geschwister Schneider USB/CAN devices and bytewerk.org candleLight USB CAN interfaces
+==================================================================================================================
```

Comment:
💯  for providing docs!

Is there any plan to support message filtering via this interface?
Are there any internal methods that users should know about? E.g. might be worth documenting the `gs_usb` instance attribute.

Please include a link to the upstream gs_usb  GitHub page (or docs)

Reply:
Definitely the docs should be updated.

Sadly the Geschwister Schneider USB/CAN and candleLight USB CAN devices do not support message filtering internally, so message filtering can not be added in this interface.

The docs is updated and more info is provided